src/Stratis.Bitcoin.Features.BlockStore/Stratis.Bitcoin.Features.BlockStore.csproj

```diff
@@ -21,12 +21,12 @@
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="DBreeze" Version="1.89.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Debug" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="3.1.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.1.0" />
-    <PackageReference Include="Tracer.Fody" Version="3.2.0">
+    <PackageReference Include="DBreeze" Version="1.95.0" />
```

Comment:
Personally I am not sure we should just upgrade all nugets to latest ver.
Did you try to run networks?

Reply:
I did, and have not run into any issues. I've run x42 for about a month with this new version and it's stable.

Reply:
ah ok thats good news then, I suggest if we can wait til my coindb refactor goes in, then I will try out the changes in this PR together with the optimization PR to sync BTC and STRAT.